src/main/webapp/app/core/interceptor/notification.interceptor.ts

```diff
@@ -25,11 +25,11 @@ export class NotificationInterceptor implements HttpInterceptor {
           }
 
           if (alert) {
-            this.alertService.addAlert({
+            /*  this.alertService.addAlert({
```

Comment:
Is this still commented out from testing?
I don't think we should use comments to disable the alert service.
If we don't need it we could remove it instead to prevent dead code.